doc/howto/network_bridge_resolved.md

```diff
@@ -70,6 +70,60 @@ You must repeat the commands after each reboot and after LXD is restarted, or ma
 
 ## Make the `resolved` configuration persistent
 
+### Solution "A"
+
+You can automate the `systemd-resolved` DNS configuration, so that it is applied on system start and takes effect when LXD creates the network interface.
+
+To do so, create a `systemd` network file named `/etc/systemd/network/<network_bridge>.network` with the following content:
+
+```
+[Match]
+Name=<network_bridge>
+[Network]
+DNS=<dns_address>
+Domains=~<dns_domain>
+```
+
+So for example, `/etc/systemd/network/lxdbr0.network` with the following content (your IP, of course):
+
+```
+[Match]
+Name=lxdbr0
+[Network]
+DNS=10.167.146.1
+Domains=~lxd
+```
+
+If you have NOT freshly installed `lxd` (you've rebooted since install), then you only need to reload `systemd-resolved`:
+
+    systemctl restart systemd-resolved.service
+
+In case you **haven't rebooted yet since first initializing `lxd`**, then you need to either A) reboot the system, or B) reload `systemd-networkd` (to reload the `.network` files) and restart `lxd` (to add the routing).
```

Comment:
```suggestion
If you have _not_ rebooted your system since you first installed LXD, you must either:
1. reboot the system, or 
1. reload `systemd-networkd` (to reload the `.network` files) and restart `lxd` (to add the routing):
```